ui/src/components/facets/HistogramFacet.js

```diff
@@ -88,6 +95,35 @@ class FacetHistogram extends Component {
       !this.props.selectedValues.includes(facetValue.name)
     );
   }
+
+  onClick(event, item) {
+    // Ignore clicks which are not located on histogram
+    // bars.
+    if (item != null) {
+      // facetValue is a string, eg "female"
+      const facetValue = item.datum.facet_value;
+      let isSelected;
+      if (
+        this.props.selectedValues != null &&
```

Comment:
// this.props.selectedValues contains what was selected before the click.
// isSelected contains if facet value was selected after the click.

Reply:
Done.